### PR TITLE
Ansible installations/configurations for access-ca deployment

### PR DESCRIPTION
**Pull request by stevelaskaridis**
## Description

Ansible directory for automated deployment of the application in either development or production environments.
## Goal

The aim of this task is to easily configure the infrastructure running the access-ca-service on demand. This saves both time and repetitiveness actions of installing packages to multiple nodes.
## Requirements
- Python installed in the target machine
- SSH access to the target machine

### CODE REVIEW

File: ansible/roles/ruby/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- name: Install rvm and ruby {{ ruby_rvm_version }}
+  shell: gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 && \curl -sSL https://get.rvm.io | bash -s stable --ruby={{ ruby_rvm_version }}
+  args:
+    creates: /usr/local/rvm/bin/rvm
+    tags:
+        - rvm_install
+        - ruby_install
+        
+- name: Register ruby version
+  shell: ruby --version | awk -F " " '{ print $2 }' | awk -F "p" '{print $1}'
```
Comment (NicolasLiampotis) [inline ansible/roles/ruby/tasks/main.yml:12]: This always seem to fail with `ruby: command not found`. Debug logs follow:

```
 rm -rf "/root/.ansible/tmp/ansible-tmp-1445330571.96-270171811862497/" > /dev/null 2>&1                                                                        
changed: [localhost -> localhost] => {"changed": true, "cmd": "ruby --version | awk -F \" \" '{ print $2 }' | awk -F \"p\" '{print $1}'", "delta": "0:00:00.096192", "end": "2015-10-20 08:42:52.172670", "rc": 0, "start": "2015-10-20 08:42:52.076478", "stderr": "/bin/sh: ruby: command not found", "stdout": "", "stdout_lines": [], "warnings": []}
```

File: ansible/roles/ruby/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  ignore_errors: True
+  register: ruby_version
```
Comment (NicolasLiampotis) [inline ansible/roles/ruby/tasks/main.yml:14]: Why do we register the `ruby_version` variable?

**Comment by stevelaskaridis** [comment]
@NicolasLiampotis: On a brand new vm with CentOS 7

```
TASK: [ruby | Register ruby version] ******************************************
changed: [snf-681691.vm.okeanos.grnet.gr] => {"changed": true, "cmd": "ruby --version | awk -F \" \" '{ print $2 }' | awk -F \"p\" '{print $1}'", "delta": "0:00:00.021351", "end": "2015-10-20 06:53:19.215371", "rc": 0, "start": "2015-10-20 06:53:19.194020", "stderr": "", "stdout": "2.2.3", "warnings": []}
```

**Comment by stevelaskaridis** [comment]
The 2nd one was in order to have a dynamic variable for the ruby version, instead of the statically typed one inside **roles/common/vars/main.yml**.
We could potentially add a test to see if the two ruby variables are the same. (the one we have statically the dynamic one)

**Comment by NicolasLiampotis** [comment]
@stevelaskaridis: `ruby --version` is failing for me on a clean Docker-based CentOS 7 installation with ansible installed. The problem is that `ruby` is not in the root user's path, although I can see it has been installed under `/usr/local/rvm/rubies/ruby-2.2.3/bin/ruby`. Anyway, I'm commenting out this task as the registered variable is not currently being used.

**Pull request merged**